meson.build

```diff
@@ -639,132 +710,77 @@ else
     missing_types += 'struct mount_attr (sys/mount.h)' endif
 
 ## Check if sys/mount.h defines the fsconfig commands
-if cc.get_define('FSCONFIG_SET_FLAG', prefix: decl_headers) != ''
+if cc.get_define('FSCONFIG_SET_FLAG', prefix: decl_headers) != '' or \
+   cc.has_header_symbol('sys/mount.h','FSCONFIG_SET_FLAG')
```

Comment:
Please, can you explain why `cc.get_define('FSCONFIG_SET_FLAG', prefix: decl_headers) != ''` is not enough here?
As far as I can see:
```
decl_headers = '''
#include <sys/mount.h>
'''
```

Have you practically checked that `cc.get_define('FSCONFIG_SET_FLAG', prefix: decl_headers) != ''` is false, while `cc.has_header_symbol('sys/mount.h','FSCONFIG_SET_FLAG')` is true? In which circumstances this happens?

My wild guess would be that we need to do something like `cc.get_define('FSCONFIG_SET_FLAG', prefix: decl_headers, args: '-D_GNU_SOURCE') != ''`.

Reply:
Can we use `cc.has_header_symbol` without `cc.get_define`? According to the documentation https://mesonbuild.com/Reference-manual_returned_compiler.html#compilerhas_header_symbol `cc.has_header_symbol` should be more universal and there is no need for `cc.get_define`. WDYT?

Sorry for being picky here, but I just want to keep meson files as simple as possible ;)